src/object_store.cpp

```diff
@@ -116,6 +117,12 @@ void insert_column(Group& group, Table& table, Property const& property, size_t
         table.insert_column_link(col_ndx, is_array(property.type) ? type_LinkList : type_Link,
                                  property.name, *link_table);
     }
+    else if (is_array(property.type)) {
+        DescriptorRef desc;
+        table.insert_column(col_ndx, type_Table, property.name, &desc);
+        desc->add_column(to_core_type(property.type & ~PropertyType::Flags), "!ARRAY_VALUE",
```

Comment:
Since we're using this magic column name in a few different places, it'd be worth pulling it into a named constant.